backend/app/thumbnails.py

```diff
@@ -0,0 +1,142 @@
+import geopandas
```

Comment:
PP: `thumbnails.py` might make more sense in `app/core` than `app` for code organization purposes; please move the file and adjust imports

Reply:
👍